packages/web/src/common/store/profile/fetchTopTagsSaga.ts

```diff
@@ -0,0 +1,27 @@
+import { getContext, profilePageActions } from '@audius/common'
+import { takeLatest, call, put } from 'typed-redux-saga'
+const {
+  FETCH_TOP_TAGS,
+  fetchTopTags,
+  fetchTopTagsSucceeded,
+  fetchTopTagsFailed
+} = profilePageActions
+
+export function* watchFetchTopTags() {
+  yield* takeLatest(FETCH_TOP_TAGS, fetchTopTagsWorker)
+}
+
+function* fetchTopTagsWorker(action: ReturnType<typeof fetchTopTags>) {
```

Comment:
I like the pattern of appending `Worker` to the name here, pretty annoying when name collisions happen with the action itself

Reply:
thanks! yeah i noticed the param name for second arg to "takeEvery" is "worker", so thought we could go with it